### DISCUSSION TITLE
dependsOn filterable is ineffective on the list page.

### DISCUSSION THREAD

**Discussion by niefufeng** [laravel/nova-issues#6376] [Ideas & Feature Requests]
- Laravel Version: 11.7
- Nova Version: 4.33.3
- PHP Version: 8.3.6
- Database Driver & Version:
- Operating System and Version: mac os 14.4.1
- Browser type and version: google chrome 124.0.6367.119
- Reproduction Repository: https://github.com/laravel/nova-issues

### Description:
dependsOn filterable is ineffective on the list page.

### Detailed steps to reproduce the issue on a fresh Nova installation:
```php
BelongsTo::make('Brand', 'brand', MaterialBrand::class)->filterable(),
BelongsTo::make('Category', 'category', MaterialCategory::class)
    ->dependsOn(
        'brand',
        function (BelongsTo $field, NovaRequest $request, FormData $formData) {
            $field->relatableQueryUsing(
                fn(NovaRequest $novaRequest, Builder $builder) => $builder->where('brand_id', $formData->integer('brand'))
            );
        }
    ),
```

**Comment by crynobone** [comment]
This is not something that Dependent Fields are designed for and would be considered.
